libs/shared/src/containers/AccountMap/reducer.ts

```diff
@@ -24,41 +25,31 @@ export class AccountMap extends BaseReducer<IAccountStoreAPI, AccountMapState>('
       if (this.state.map[address] === undefined)
         throw new Error(`Cannot set activeAccount: ${address} does not exist`);
     }
-
+    log.trace({address}, 'Set active to {address}');
     this.draftState.active = address
   }
 
-  // Add a new account, optionally store in LocalStorate, in unlocked state
-  addAccount(name: string, address: string, signer: Signer, store: boolean = true, setActive: boolean = true, unlocked?: Wallet) {
-    // Check the signer & unlocked are actually the same account
-    if (unlocked && NormalizeAddress(address) != NormalizeAddress(unlocked?.address)) {
-      throw new Error("Accounts being stored are mis-matched");
-    }
+  // Add a new account
+  addAccount(name: string, address: string, signer: Signer) {
```

Comment:
It turned out the code sending our argumenst stripped optional arguments.  How did this ever work???